pkg/cmd/server/server.go

```diff
@@ -501,6 +514,15 @@ func (s *server) runControllers(config *api.Config) error {
 		s.logger,
 	)
 
+	go func() {
+		metricsMux := http.NewServeMux()
+		metricsMux.Handle("/metrics", promhttp.Handler())
+		err := http.ListenAndServe(s.metricsAddress, metricsMux)
+		s.logger.Fatalf("Failed to start metrics: %v", err)
```

Comment:
Let's only make the fatal call if err != nil

Reply:
Can you please also verify that if you run `ark server` by hand and you do `ctrl-c` that it shuts down cleanly and this doesn't keep it running?

Reply:
oops! Thanks for catching that.